builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -196,18 +196,26 @@ func respondReadIssuer(issuer *issuerEntry) (*logical.Response, error) {
 		revSigAlgStr = ""
 	}
 
+	data := map[string]interface{}{
+		"issuer_id":                      issuer.ID,
+		"issuer_name":                    issuer.Name,
+		"key_id":                         issuer.KeyID,
+		"certificate":                    issuer.Certificate,
+		"manual_chain":                   respManualChain,
+		"ca_chain":                       issuer.CAChain,
+		"leaf_not_after_behavior":        issuer.LeafNotAfterBehavior.String(),
+		"usage":                          issuer.Usage.Names(),
+		"revocation_signature_algorithm": revSigAlgStr,
+		"revoked":                        issuer.Revoked,
+	}
+
+	if issuer.Revoked {
+		data["revocation_time"] = issuer.RevocationTime
+		data["revocation_time_utc"] = issuer.RevocationTimeUTC
```

Comment:
I remember Nick asking at one point if we could return a human readable form, might be a nice thing here? Or should we tackle that in another PR across all the various revocation times perhaps?

Reply:
Isn't the `_utc` variant human readable, ish?

```
revocation_time                   1660656936
revocation_time_utc               2022-08-16T13:35:36.782087754Z

    "revocation_time": 1660656936,
    "revocation_time_utc": "2022-08-16T13:35:36.782087754Z",
```

Reply:
Ah you are correct, `revocation_time_utc` is missing from the sample response section in the doc update which is what originally triggered this thought.

Reply:
Ah gotcha. Yeah, it wasn't in revoke, so we should probably add it sometime.